.github/workflows/deliverMainBranch.yml

```diff
@@ -0,0 +1,40 @@
+# This workflow will build a Java project with Maven, and cache/restore any dependencies to improve the workflow execution time
+# For more information see: https://docs.github.com/en/actions/automating-builds-and-tests/building-and-testing-java-with-maven
+
+# This workflow uses actions that are not certified by GitHub.
+# They are provided by a third-party and are governed by
+# separate terms of service, privacy policy, and support
+# documentation.
+
+name: Deliver main branch
+
+on:
+  push:
+    branches: [ "main" ]
+  pull_request:
+    branches: [ "main" ]
+
+jobs:
+  build:
+
+    runs-on: ubuntu-latest
+
+    steps:
+    - uses: actions/checkout@v3
+    - name: Set up JDK 17
+      uses: actions/setup-java@v3
+      with:
+        java-version: '17'
+        distribution: 'temurin'
+        cache: maven
+    - name: Build with Maven
+      run: mvn -B package --file pom.xml
```

Comment:
Since the action results into an error (file not found), I think the path has to be adjusted here since the pom.xml file can be found only under /Maintenance Monitor/pom.xml

Reply:
File path is corrected and workflow action run has been successful. Build was done and an artifact was created: the Build.zip archive can be downloaded containing the executable .jar-file.

Reply:
looks good to me